### DISCUSSION TITLE
Sol to Eth bridge

### DISCUSSION THREAD

**Discussion by jess82494** [wormhole-foundation/wormhole#530] [Problems / bugs] [answered]
I tried to bridge from Phantom to Metamask and my SOL/ETH is gone :((
TX ID:2ewj244VzKCxBoTVfdK2MwChd37u4tTQvwUrgbXgmcoiT6W6pGfiJW7Ch9S4cSNB1WhcF1KB4XpXfFFg5SYNyPhj
I used the recovery dialogue feature and it showed this error message:cannot estimate gas; transaction may fail or may require manual gas limit (error={"code":-32603,"message":"execution reverted: transfer already completed","data":{"originalError":{"code":3,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001a7472616e7366657220616c726561647920636f6d706c65746564000000000000","message":"execution reverted: transfer already completed"}}}, method="estimateGas", transaction={"from":"0x3E6a649DcB7835923FCE4A530A18671fd294f953","to":"0x3ee18B2214AFF97000D974cf647E7C347E8fa585","data":"0xc6878519000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000004de01000000011000928cbacd9035b4bfb9e8ff5cb942f95c941923e844f6c4c814a4033ed12858c872094ceed886c8193d896dfc1f221d70adf80243e6b1af248670e29ecb1bc2e300019419a2cf65869f1815d2c4b5bfbcfe88d3837209b536a9eaa01f8ce028da7fd95000fcfb7c64e823a3017106a5d700c5571d45060b86d6f288095aef0f704d3101025680d86103b1d59861e778830df68b30d4cf72edc0c4112b6355df9647769cb8252872f6319d209c1ea6c7ad9743e4a0056e5e92c50a7b492ab06cd29964fa4a01031d58b4377349784a36923d10438871a07fdb72192c757f9cf30a96125b286cdb13511725ba7707a9a6d371c3a008df0af789d90f9b78a668109529fb72e636af010415aacfe334a2ebbc9527d97a56d8c14a04bfababdb807425c3720674cb8f4cab4a6a4de8e9a59bba9c48ddad1e0ce8755e283d9587f6cee5210bdd762e690eb40005db6491a55577fc98460598e6f4c43ac5ee3902ea7a12859d87c1f1cf4029f4ee5f174f3eaca33997673a3e39551fa5b6795a2d7ce50cbf94f1a4bf91706bb14001067a2a5fa2bbbe96ef55f5e51ab478ea5f7957177645cfde03bcb6e67b2ffa49154054b7c99cfed6e372760f57d6d9b930dd2c33c13fc82928bb685394af83e8540107f1c13c83f54007c5f1420809b7b4b060b046891b04697257ab14d1cb08d081162427494e63f4dedf39fd186845ea4de711f4f321c8bf0356f714a57a46498b9f00093ba35cb0e59532deca30de0ef87abe9ef228c8779004e8fba6cdf9b93f3941585578fd9791f3c999e6c4126f7b8a0b6e2ba523bd539f0a3f7a68fd4f32e21fd3000aeb12a307bd420b79ed9fb3f292889411d5ff5267e98d1a809c1a59f0dd40d77434e5cba94268a9010d800220b3db988996f81a118f4d32d68fe24bde6f8f5ab7000cb45817113e68f82fccad5484d1626161dd28d6f357a1c3bef58577add9bba43546c9ad3e577aebbf2eedfd123d36f26b6ec4134c85abdeed8aa523be5a6cc89c000d3841410009d9361cd73fcc06f9cf8d45b58db6f8ef6e52334333bb2043f3c40c6e9a900c0e5c583ed9b21af794e38531c633e20614e906509ce2acf4ef85e3ff000eef591d9cab52678487b6f384197c0d96f83894ca85a5f18fa14cf35bf0cb2eaf0d46cc3a6db27db64ade9407ffd08d8d713c0ad8b46652572b7d8211b1bd8ce9010f1e85b17f02573274780ca78c27e205a79aae169d623482efb9178d81ab98e105754f3f73cb69db7f23c414c2ff26303e4ff39adc2baaf61673eb67ac02ed81dc0010a809bd9160d516cf2f9fa1ccef791086f6d7414f51296f4de76fd55ae5af432e595af1ffa1d5dbf6636748340b0f85f3d2ba1a3aff55a643e87638e3b41cab2b0111d7bcedc9a1c22543cbca4cdd79bf7be4ff7bf347dc47d21bc8c2ea9f2d6d8ce11a5d203c555ad4ed170f0052b72f96eb70a63a92e17eb4ac6b37336428f949c000615e0d9d000117bd0001ec7372995d5cc8732397fb0ad35c0121e0eaa90d26f828a534cab54391b3a4f500000000000006592001000000000000000000000000000000000000000000000000000000003b9aca00069b8857feab8184fb687f634618c035dac439dc1aeb3b5598a0f0000000000100010000000000000000000000003e6a649dcb7835923fce4a530a18671fd294f953000200000000000000000000000000000000000000000000000000000000000000000000","accessList":null}, code=UNPREDICTABLE_GAS_LIMIT, version=providers/5.4.5)

**Comment by SEJeff** [comment]
@jess82494 please ignore the advise of @Stevesol97 who is a scammer trying to empty your wallet via his scam website. The developers of wormhole will help you get this sorted.

**Comment by leoluk** [comment, marked as answer]
The `execution reverted: transfer already completed` error suggests that the transfer was, in fact, completed.

**Reply by tamingpalling** [reply]
hello I have the same problem is there any way I could get some help?

**Reply by sleepycorgi1** [reply]
@tamingpalling would you be able to join the wormhole discord? I can assist you there, we have a support channel under #wormhole-support-v2 
https://discord.gg/P4fAJGFe 
Thanks! SS
